tests/SimpleStringTest.cpp

```diff
@@ -405,7 +411,7 @@ TEST(SimpleString, findNormal)
     SimpleString str("Hello World");
     LONGS_EQUAL(0, str.find('H'));
     LONGS_EQUAL(1, str.find('e'));
-    LONGS_EQUAL(-1, str.find('!'));
+    LONGS_EQUAL(SimpleString::npos, str.find('!'));
```

Comment:
The benefit is using size_t where it should be used, and not mixing it with int (see further up in the code).

If you really don't want this, I'll do another PR just with the method.

I can also do
```c++
    SimpleString subString(size_t beginPos, size_t amount = size()) const;
```
Why would you prefer a second method?